macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -100,7 +102,9 @@
         bucket_start,
         bucket_end,
         bucket_duration_hours,
-        {{- dbt_utils.current_timestamp_in_utc() -}} as updated_at
+        {{- dbt_utils.current_timestamp_in_utc() -}} as updated_at,
+        dimension,
+        dimension_value
```

Comment:
Shouldn't dimension and dimension value impact the id? 
If I have two dimension tests on the same table with two different dimensions, they will override each other